app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestDeletePet.java

```diff
@@ -0,0 +1,54 @@
+package org.pesmypetcare.mypetcare.features.users;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.controllers.TrDeletePet;
+import org.pesmypetcare.mypetcare.features.pets.Gender;
+import org.pesmypetcare.mypetcare.features.pets.Pet;
+import org.pesmypetcare.mypetcare.features.pets.PetRepeatException;
+import org.pesmypetcare.mypetcare.features.pets.UserIsNotOwnerException;
+import org.pesmypetcare.mypetcare.services.StubPetManagerService;
+
+import static org.junit.Assert.assertFalse;
+
+public class TestDeletePet {
+    private User user;
+    private Pet pet;
+    private TrDeletePet trDeletePet;
+
+    @Before
+    public void setUp() throws PetRepeatException {
+        user = new User("johnDoe", "albert69@gmail.com", "1234");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "albert69@gmail.com" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5232056)